static/scss/partials/main.scss

```diff
@@ -1967,7 +1967,7 @@ input:focus::placeholder {
     justify-content: center;
     align-items: center;
     display: flex;
-    margin-left: 3em;
+    margin-left: $layout-md;
```

Comment:
Opted for `$layout-md` instead of spacing, since it involves the placement of an element relative to another.